### PR TITLE
[CU-3vdqb2b] Improve locked value storing and lazily calculate total locked value

### PR DESCRIPTION
**Pull request by andresmechali**
## Issue

https://app.clickup.com/t/3vdqb2b

This requires reprocessing.

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-3vdqb2b Fix daily active users chart (Subsquid)](https://app.clickup.com/t/3vdqb2b)

File: subsquid/src/server-extension/resolvers/picassoOverviewStats.ts [hunk 1]
```diff
@@ -36,26 +44,45 @@ export class PicassoOverviewStatsResolver
...
 
-    if (!lockedValue?.[0]) {
-      return Promise.resolve(0n);
+    for (const assetId of Object.keys(assetPrices)) {
+      try {
+        const oraclePrice = await api.query.oracle.prices(assetId);
+        if (oraclePrice?.price) {
+          assetPrices[assetId] = BigInt(oraclePrice?.price?.toString() || 0n);
+        }
+        // TODO: what to do if no there's no Oracle price?
```
Comment (dzmitry-lahoda) [inline subsquid/src/server-extension/resolvers/picassoOverviewStats.ts:72]: return "N/A"

**Review by dzmitry-lahoda — APPROVED** [review]

File: subsquid/src/server-extension/resolvers/picassoOverviewStats.ts (hunk 1 above)
Comment (andresmechali) [reply subsquid/src/server-extension/resolvers/picassoOverviewStats.ts]: In the case of this resolver it's accumulating over all the locked assets, so if one locked asset is missing its price, I can only think on ignoring it, __unless__ there's some other source I can get prices from.

**Review by saadjhk — APPROVED** [review]

**Comment by andresmechali** [comment]
@mergify refresh

**Pull request merged**